### PR TITLE
XCode FLAMEX 

### PR DESCRIPTION
**Pull request by JasonKnights**
If game doesn't accpet this, game is gay

### CODE REVIEW

File: AboutUs.swift
Comment (noigamegun) [inline AboutUs.swift:1]: You be credited in the repo. no need for this

File: Media.xcassets/Jason.imageset/PictureJason.jpg
Comment (noigamegun) [inline Media.xcassets/Jason.imageset/PictureJason.jpg:1]: You be credited in the repo. no need for this

File: Media.xcassets/Jason.imageset/Contents.json
Comment (noigamegun) [inline Media.xcassets/Jason.imageset/Contents.json:1]: You be credited in the repo. no need for this

**Review by GamuGamiChan — CHANGES_REQUESTED** [review]
You be credited in the repo. no need for this

**Comment by noigamegun** [comment]
I'll go try to fix the conflits

File: MainMenu.swift
Comment (noigamegun) [inline MainMenu.swift:1]: Dumb Typo Lmao

**Review by GamuGamiChan — CHANGES_REQUESTED** [review]

**Comment by noigamegun** [comment]
Wrong Branch. I said that use the "latestmain" branch not the "new" branch

**Comment by noigamegun** [comment]
you can transfer your changes in "new" branch to "latestnew" branch

> If game doesn't accpet this, game is gay
well you don't know git lmao

**Pull request closed without merging**